src/components/event/EventWindow.tsx

```diff
@@ -22,12 +22,34 @@ import '../../styles/events.scss';
 import { useEventWindowViewStore } from '../../hooks/useEventWindowViewStore';
 import FlatEventView from './flat-event-list/FlatEventView';
 
-const EventWindow = () => {
+interface EventWindowProps {
+	isActive: boolean;
+	setIsActive: () => void;
+}
+
+const EventWindow = (props: EventWindowProps) => {
 	const viewStore = useEventWindowViewStore();
+	const { isActive, setIsActive } = props;
+	const ref = React.useRef<HTMLDivElement | null>(null);
+
+	const documentClickHandler = (e: MouseEvent) => {
+		const tabs = ref.current && ref.current.closest('.tabs');
```

Comment:
if you do something jquery style that means something went wrong. I think we should move this click listener to AppWindow actually. Because in this component we dont have access to tabs. We also can check if tab is already active then we don't need to set it active again.